docs/guide/adapters/emberznet.md

```diff
@@ -112,7 +112,38 @@ With external antenna.
 
 
 ### Hybrid (USB + Network)
+<details>
+<summary>SMLIGHT SLZB-06M EFR32MG21 Zigbee Ethernet USB POE WiFi LAN adapter</summary>  
```

Comment:
Can you make this entire section more similar to the other adapters? A short description + link to product page and shop.

Reply:
@Koenkk done!

Reply:
https://github.com/Koenkk/zigbee2mqtt.io/pull/2747/commits/d1cdf9d8625ca0eabb527618212ca620439007a1

Reply:
It still contains more than just a link to product page + buy link. E.g. everything below `- Zigbee firmware:  `

Reply:
> It still contains more than just a link to product page + buy link. E.g. everything below `- Zigbee firmware: `

@Koenkk got it. understood. done here: https://github.com/Koenkk/zigbee2mqtt.io/pull/2747/commits/bf03ef9b00c32ddb1114237d97062ff3a80983a6